eva-accession-release-automation/run_release_in_embassy/run_release_for_species.py

```diff
@@ -13,185 +13,125 @@
-                                                                                             assembly_accession)
-            workflow_report_file_name = workflow_file_name.replace(".nf", ".report.html")
-            if os.path.exists(workflow_report_file_name):
-                os.remove(workflow_report_file_name)
-            workflow_command = "cd {0} && {1} run {2} -c {3} -with-report {4} -bg".format(
-                os.path.dirname(release_log_file),
-                common_release_properties["nextflow-binary-path"], workflow_file_name,
-                common_release_properties["nextflow-config-path"], workflow_report_file_name)
-            logger.info("Check log file in: " + release_log_file + " to monitor progress...")
-            logger.info("Running workflow file {0} with the following command:\n\n {1} \n\n"
-                        "Use the above command with -resume if this workflow needs to be resumed in the future"
-                        .format(workflow_file_name, workflow_command))
-            os.system(workflow_command)
+            nextflow_params = get_nextflow_params(taxonomy_id, assembly_accession, release_version)
+            workflow_file_path = get_run_release_for_assembly_nextflow()
+            release_dir = get_assembly_release_folder(taxonomy_id, assembly_accession)
+            nextflow_config = get_nextflow_config()
+            workflow_command = ' '.join((
+                f"cd {release_dir} &&",
+                f"{cfg.query('executable', 'nextflow')} run {workflow_file_path}",
+                f"-params-file {nextflow_params}",
+                f'-c {nextflow_config}' if nextflow_config else '',
+                '-resume' if resume else '',
+            ))
+            logger.info(f"Running workflow file {workflow_file_path} with the following command: "
+                        f"\n {workflow_command} \n")
+            run_command_with_output(f"Running workflow file {workflow_file_path}", workflow_command)
+
+
+def load_config(*args):
+    cfg.load_config_file(
+        *args,
+        os.environ.get('RELEASE_CONFIG'),
+        '~/.release_config.yml'
+    )
 
 
 def main():
     argparse = ArgumentParser()
-    argparse.add_argument("--common-release-properties-file", help="ex: /path/to/release/properties.yml", required=True)
-    argparse.add_argument("--taxonomy-id", help="ex: 9913", required=True)
-    argparse.add_argument("--assembly-accessions", nargs='+', help="ex: GCA_000003055.3")
+    argparse.add_argument("--taxonomy_id", help="ex: 9913", required=True)
+    argparse.add_argument("--assembly_accessions", nargs='+', help="ex: GCA_000003055.3")
+    argparse.add_argument("--release_version", required=True)
+    argparse.add_argument("--resume", default=False, required=False,
+                          help="Resume the nextflow pipeline for the specified taxonomy and assembly")
+    argparse.add_argument("--release_config_file",
```

Comment:
Do you think this is needed? I think elsewhere, if the config can be set by an environment variable we don't have command-line argument for it as well, but maybe you have a use case in mind.

Reply:
I was mostly mimicking the previous behaviour. You're right it is not required